youtube_dl/extractor/xvideos.py

```diff
@@ -17,7 +17,8 @@ class XVideosIE(InfoExtractor):
     _VALID_URL = r'''(?x)
                     https?://
                         (?:
-                            (?:www\.)?xvideos\.com/video|
+                            (?:(?:fr|it|de|www)\.)?xvideos2?\.com/video|
```

Comment:
There are more country domains.

Reply:
>There are more country domains

When I wrote the regex, I used the html code from the https://xvideos.com main page. A part from 'fr', 'it' and 'de' xvideos.com subdomains and xvideos.es there were no other country domains.
All other countries lead to www.xvideos2.com.

Here is an excerpt from this html code (as of 2019-09-28)

    <link rel="alternate" href="https://www.xvideos.com/" hreflang="x-default" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="en-IN" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="hi" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="bn" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="mr" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="te" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="ta" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="kn" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="gu" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="or" />
    <link rel="alternate" href="https://www.xvideos2.com/" hreflang="ml" />
    <link rel="alternate" href="https://www.xvideos.es/" hreflang="es" />
    <link rel="alternate" href="https://fr.xvideos.com/" hreflang="fr" />
    <link rel="alternate" href="https://it.xvideos.com/" hreflang="it" />
    <link rel="alternate" href="https://de.xvideos.com/" hreflang="de" />

As of this writing, https://www.xvideos.cm/ exists. However it seems to be a chaturbate like website. xvideos.desi is a parked domain.

@dstftw Do you see any other country domain needing to be added?

Reply:
jp also exists